lib/rust/ensogl/component/grid-view/src/header.rs

```diff
@@ -528,6 +529,46 @@ where
     }
 }
 
+impl<Entry, InnerGridView, HeaderEntry, HeaderModel, HeaderParams> FrpNetworkProvider
+    for GridViewTemplate<Entry, InnerGridView, HeaderEntry, HeaderModel, HeaderParams>
+where
+    InnerGridView: FrpNetworkProvider,
+    HeaderModel: frp::node::Data,
+{
+    fn network(&self) -> &frp::Network {
+        self.model.grid.network()
+    }
+}
+
+impl<E: Entry, HeaderEntry: Entry<Params = E::Params>> application::View
```

Comment:
Why only header version implements application::View? It should be implemented by selectable variant (with headers or without).